File: strlen.java
import java.util.*;
import java.lang.*;
import java.io.*;


public class strlen{
	public static int Len(String s){
		if(s.equals("")) return 0;
		else return 1 + Len(s.substring(1));
	}
	public static void main(String[] args) {
		String s = "ABCDEFG";
		System.out.println(Len(s));
	}
}

File: strlen.py




def Strlen(s):
	if s == "":
		return 0
	else:
		return 1 + Strlen(s[1:])


def Fibonacci(n):
	if n == 0 or n == 1:
		return 1
	else:
		return Fibonacci(n-1) + Fibonacci(n-2)

s = "abcdefg"

#print(Strlen(s))
print(Fibonacci(5))



